db/migrate/20150525081931_remove_duplicate_tokens.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveDuplicateTokens < ActiveRecord::Migration
+  def up
+    existing_tokens = Host::Managed.joins(:token).map(&:token).compact
+    if existing_tokens.empty?
+      Token.delete_all
+    else
+      Token.where('id not in (?)', existing_tokens).delete_all
+    end
+
+    remove_index :tokens, :host_id
```

Comment:
thanks, I'll try to check if it exists but it looks like it would happen even if the index exists `Mysql2::Error: Cannot drop index 'index_tokens_on_host_id': needed in a foreign key constraint`